flow_judge/models/adapters/baseten/adapter.py

```diff
@@ -0,0 +1,169 @@
+import os
+import json
+import time
+import requests
+
+from typing import Any, Dict
+
+from ..base import BaseAPIAdapter
+
+class BasetenAPIAdapter(BaseAPIAdapter):
```

Comment:
This looks to me like 1 to 1 implementation of the openai client library for Python which would offer offloading the maintenance cost to them and benefit from their error handling and string encoding to all platforms.